draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -12115,7 +12115,8 @@ Content-Encoding: gzip
 <x:ref>URI-reference</x:ref> = &lt;URI-reference, see <xref target="RFC3986" x:fmt="," x:sec="4.1"/>&gt;
 <x:ref>User-Agent</x:ref> = product *( RWS ( product / comment ) )
 
-<x:ref>Vary</x:ref> = "*" / ( field-name *( OWS "," OWS field-name ) )
+<x:ref>Vary</x:ref> = *( "," OWS ) ( "*" / field-name ) *( OWS "," [ OWS ( "*" /
+ field-name ) ] )
```

Comment:
This bit is wrong -- is that because you haven't updated bap yet?

Reply:
very possibly; will try again.

Reply:
Hm, I re-made and it didn't change. @reschke ?